src/components/Link.tsx

```diff
@@ -113,6 +115,12 @@ const Link: React.FC<IProps> = ({
         {...commonProps}
       >
         {children}
+        <VisuallyHidden>(opens in a new tab)</VisuallyHidden>
+        {!hideArrow && (
+          <Box as="span" ml={0.5} mr={1.5} aria-hidden>
```

Comment:
The addition of the `VisuallyHidden` content is great!

I'm still having the initial problem that this set out to solve, the arrow is being read out by the screen reader.

This is the DOM structure being created, so I think it has to do with the children still being accessible by the screen reader.
<img width="864" alt="Screen Shot 2023-03-22 at 12 44 33 PM" src="https://user-images.githubusercontent.com/15589226/227006282-3259cee5-ae3d-420c-9019-b2243fa7fe81.png">

Will look a bit into how to resolve this.

Reply:
@corwintines hmmm what screen reader did you use?

Reply:
@TylerAPfledderer this is what I have at the moment. https://getpericles.com/

If this is a screen reader issue, I can look for a new one.